docs/operate-and-deploy/migrations-tool.md

```diff
@@ -128,7 +128,7 @@ secure ksqlDB servers, such as credentials for HTTP basic authentication or TLS
 and truststores, as well as optional configurations specific to the migrations tool.
 
 See the [config reference](#config-reference) for details on individual configs.
-See [here](#connecting-to-confluent-cloud-ksqldb) for the configs required to 
+See [here](#connecting-to--siteccloud--ksqldb) for the configs required to 
```

Comment:
I think this needs to be the expanded value `#connecting-to-confluent-cloud-ksqldb` because the title tag below will have the variable expanded.

Reply:
Thanks for the catch. It's too bad that means the link is broken when viewed from Github but that's minor as long as the link works on the live site :)